src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A printable interface for a Person in the addressbook.
```

Comment:
This interface need not be specific to addressbook. Hence, you may want to update the header comment here.